flatcar-postinst

```diff
@@ -244,6 +244,14 @@ if mountpoint -q /etc; then
   unshare -m sh -c "umount /etc && mkdir -p /etc/extensions && attr -R -r overlay.opaque /etc/extensions || true"
 fi
 
+# Systemd version >= 256 does not boot anymore if cgroupv1 is enabled or SYSTEMD_CGROUP_ENABLE_LEGACY_FORCE=1 is set
+# See: https://github.com/systemd/systemd/releases/tag/v256-rc3
+MAX_NEXT_VERSION_ID=5000
+if [ "${NEXT_VERSION_ID%%.*}" -ge $MAX_NEXT_VERSION_ID && ! -f "/sys/fs/cgroup/cgroup.controllers" ]; then
```

Comment:
i think we don't need a version check here. We can ensure that any Flatcar version that has an update-engine that contains this change is merged together with the systemd-256 update.

am i wrong about this?

Reply:
Removed the check, but left the version number that will be changed accordingly so the user gets the release information.